.github/ci_templates/e2e_tests.yml

```diff
@@ -98,6 +98,9 @@
   REPOSITORY: ${{ github.repository }}
   DHCTL_LOG_FILE: ${{ steps.setup.outputs.dhctl-log-file}}
   GITHUB_TOKEN: ${{secrets.BOATSWAIN_GITHUB_TOKEN}}
+{!{- if eq $script_arg "run-test" }!}
```

Comment:
You should also check for `coll.Has $ctx "manualRun"`, here and everywhere else, as it shouldn't be present in automatic daily tests